demisto_sdk/tests/test_files/valid_content_repo/IndicatorFields/field.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
why do we need all these test files?

Reply:
Basically just checked that everything was created ok.
I can remove the json files.

Reply:
lets remove